pykern/pkio.py

```diff
@@ -285,14 +271,17 @@ def save_chdir(dirname, mkdir=False, is_pkunit_prefix=False):
             pkunit_prefix = prev_ppp
 
 
-def sorted_glob(path, key=None):
-    """sorted list of py.path.Local objects
+def sorted_glob(pattern, key=None):
```

Comment:
Changing a parameter name is dangerous from a keyword arg perspective. I'll accept this, but I think it's important to note.

I was curious: glob.glob uses pathname, which is why it is just path here. C & Ruby use pattern. Go uses glob. Perl uses expr.